components/vim/Makefile

```diff
@@ -103,6 +103,8 @@ $(BUILD_DIR)/license: $(SOURCE_DIR)/runtime/doc/uganda.txt
 build:		$(BUILD_32)
 
 install:	$(INSTALL_32)
+		@cp $(COMPONENT_DIR)/vimrc $(PROTO_DIR)/usr/share/vim/vimrc
```

Comment:
Consider using variables from `make-rules/shared-macros.mk` like `$(PROTOUSRSHAREDIR)/$(COMPONENT_NAME)` to ease possible restructuring later in the project evolution ;)